src/updater.cpp

```diff
@@ -497,6 +497,16 @@ QString Updater::getPreviousVersion() const
 #if defined(Q_OS_WIN32)
 bool Updater::is64BitCompatible() const 
```

Comment:
I think a comment explaining what being "64BitCompatible" means here is needed - we are trying to detect "machines running a 32-Bit build of Mudlet on a 64-Bit Intel/AMD processor". That being the case the code that this PR adds can be reduced to merely:
```cpp
#if defined(Q_OS_WIN64)
    return true;
#endif
```
with no need to have a constant variable `is64BitBuild` whose only usage is to do the same.